tests/Microsoft.Identity.Test.Unit/RequestsTests/FociTests.cs

```diff
@@ -116,8 +116,12 @@ public async Task FociDoesNotHideRTRefreshErrorsAsync()
                 // B cannot acquire a token interactivelty, but will try to use FRT
                 var ex = await AssertException.TaskThrowsAsync<MsalUiRequiredException>(
                     () => SilentAsync(_appB, ServerTokenResponse.OtherError)).ConfigureAwait(false);
+#if NET_CORE
+                Assert.AreEqual(MsalError.NoTokensFoundError, ex.ErrorCode);
```

Comment:
Since this is using the netstandard runtime, there is a slight difference in the error response from ESTS due to netstandard 2.0 not being on the allowed list